truncate.js

```diff
@@ -72,9 +72,14 @@ function truncate(string, options) {
   if (end < 1) {
     return omission
```

Comment:
Can you tell me what this omission variable does

Reply:
it's just period character ("..."), which is added at the end of the resulting string after/if it is truncated

Reply:
Oh okay, thanks